classes/integrity.js

```diff
@@ -5,10 +5,13 @@
 
 const abslog = require('abslog');
 const { join } = require('path');
-const { schemas, ValidationError } = require('@eik/common');
+
+const schemas = require('@eik/common-schemas');
+
+const { typeSlug } = require('@eik/common-utils');
 const fetch = require('node-fetch');
-const { typeSlug } = require('../utils');
 
+const { ValidationError } = schemas;
```

Comment:
A little bit of consistency nit picking here, but I find the deconstruction here a little bit inconsistent.

This means that some places we are doing ex so:

```
schemas.assert.version(this.version);
```

and other places so:

```
throw new ValidationError(`Parameter "debug" is not valid`);
```

I would prefer that we either deconstructed both so we do as follow:

```
assert.version(this.version);
throw new ValidationError(`Parameter "debug" is not valid`);
```

or non so we do as follow:

```
schemas.assert.version(this.version);
throw new schemas.ValidationError(`Parameter "debug" is not valid`);
```